### PR TITLE
[#6089] Prevent checksums on archive replicas (4-2-stable)

### PR DESCRIPTION
**Pull request by alanking**
CI tests passed

### CODE REVIEW

**Comment by trel** [comment]
"iRODS currently takes" ... does that mean 'before this commit' ?

File: plugins/resources/compound/libcompound.cpp [hunk 1]
```diff
@@ -762,6 +762,7 @@ irods::error repl_object(
         }
 
         //ret = resc->call<const char*>( _ctx.comm(), irods::RESOURCE_OP_SYNCTOARCH, file_obj, L1desc[source_l1descInx].dataObjInfo->filePath );
```
Comment (trel) [inline plugins/resources/compound/libcompound.cpp:764]: is this just a dead line?

**Comment by alanking** [comment]
> "iRODS currently takes" ... does that mean 'before this commit' ?

Right, that was my intent. The following sentence then says "As of this change" to further emphasize, but I'm willing to reword it.

File: plugins/resources/compound/libcompound.cpp (hunk 1 above)
Comment (alanking) [reply plugins/resources/compound/libcompound.cpp:764]: Yeps, can remove

**Comment by trel** [comment]
okay, yeah, the "currently" is ambiguous in a commit message.  please to reword.

File: plugins/resources/compound/libcompound.cpp (hunk 1 above)
Comment (trel) [reply plugins/resources/compound/libcompound.cpp:764]: cool, yes please.

**Comment by alanking** [comment]
Removed dead line and reworded commit message.

**Comment by trel** [comment]
yeps, please # on green

**Comment by alanking** [comment]
#'d

**Pull request merged**